calc.rb

```diff
@@ -0,0 +1,64 @@
+# define add method
+def add(a, b)
+  return a + b
+end
```

Comment:
I like that you've split these out into separate methods! It may seem a little pedantic for things like + and -, but for more complicated methods (for example checking for zero before doing division) it results in much cleaner code.